src/tests/Common/CoreCLRTestLibrary/Utilities.cs

```diff
@@ -99,6 +99,18 @@ public static bool IsWindowsIoTCore
         public static bool IsNativeAot => IsNotMonoRuntime && !IsReflectionEmitSupported;
         public static bool IsNotNativeAot => !IsNativeAot;
 
+        public static bool IsClrInterpreterActive
```

Comment:
Can this be called just `IsInterpreterActive` to match the other property name?

I am not sure why we have this duplication, like `Utilities.IsGCStress` vs. `CoreClrConfigurationDetection.IsGCStress`. I do not expect it to be fixed in this PR.

Reply:
I feel like that name is misleading here since we have mono sensing stuff in this file, and this property does not detect the mono interpreter.

Reply:
I would call both `Utilities` and `CoreClrConfigurationDetection` properties `IsCoreClrInterpreter` then. I expect that we will need the same property defined in one more place under libraries - next to `IsMonoInterpreter`.

- `Clr` is typically used to refer to .NET Framework, so let's call it `CoreClr`
- I do not think we need "Active" in the name. The existing property under libraries is called `IsMonoInterpreter`

Reply:
I can change the Utilities one to IsCoreClrInterpreter but I can't change the other one in this PR because it lives in Arcade. Should I rename it in Arcade?

Reply:
I would rename it in Arcade